src/test/java/org/jenkinsci/plugins/durabletask/BourneShellScriptTest.java

```diff
@@ -120,27 +236,37 @@ public void smokeTest() throws Exception {
         assertTrue(log.contains("sleep 999"));
         assertTrue(log.contains("got SIG"));
         c.cleanup(ws);
+        if (simpleOrTini()) {
+            assertTrue("no zombies running", noZombies());
```

Comment:
Would be good to have a comment somewhere to explain what is special about these cases that causes them to leave zombies behind.

Reply:
Will do, adding this comment as a reminder to add a bit more in this thread as well because I actually am wondering if we still need to keep these specific zombie tests.